### DISCUSSION TITLE
Some beatmaps doesn't load: "Beatmap failed to load"

### DISCUSSION THREAD

**Discussion by Cyanoure** [ppy/osu#27469] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Some beatmaps doesn't load, it shows a "Beatmap failed to load" message.
I checked for one mentioned file, it doesn't exist.
I don't know what happened or what did I do that caused this issue, I suddenly run into a beatmap with this error. I was played it before, about 1 month ago.
It would be nice to have a button in the settings to re-download all beatmaps with just a click, or re-download beatmaps that has different checksum or does not exist.
I had the same problem yesterday, I gave a chance for the update if it fixes it, but it didn't.

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/30216303/ec53f02b-9fb5-444f-bd5c-3f40785fb2a5)

### Version

2024.302.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14469781/compressed-logs.zip)

**Comment by JeremyStarTM** [comment]
I have the same issue with many of my installed beatmaps too :(

**Comment by peppy** [comment, marked as answer]
This means you (or something on your system) deleted files from your lazer install.

This is non-recoverable. You can delete or reimport the beatmaps which aren't working or reset your lazer database and start from scartch.

**Reply by Cyanoure** [reply]
Could you please add an option to re-download "corrupted" beatmaps? If the game knows what beatmap is it, then it's not impossible to automatically re-download them.
It's a pain to manually reimport everything.
I have no idea what could have deleted the files, but they're definitely missing.
Should I open a new issue about this request?

**Reply by peppy** [reply]
There's already multiple threads requesting something like this, but yes, we may add this eventually.
